hermes-common/src/main/java/pl/allegro/tech/hermes/domain/topic/schema/CompiledSchema.java

```diff
@@ -0,0 +1,48 @@
+package pl.allegro.tech.hermes.domain.topic.schema;
+
+import java.util.Objects;
+
+public class CompiledSchema<T> {
```

Comment:
Yes it might look a little weird. In this case compiled schema means parsed from String object into something useful (like Schema or JsonSchema). We called this process 'schema compilation' in the first iteration of introducing schema validation to hermes. There is a bunch of classes that is following this naming convention, most notably SchemaCompilersFactory and SchemaCompiler. I agree that it might be questionable whether it should be called 'compilation' but at this point we either rename entire process with all classes involved or follow existing convention.